### PR TITLE
Update feedback for Beta website launch

### PR DESCRIPTION
**Pull request by simonrjones**
Suggested edits to help gather feedback for the Beta website

### CODE REVIEW

**Comment by simonrjones** [comment]
Fixes #204

File: README.md [hunk 1]
```diff
@@ -1,13 +1,35 @@
...
+
+The [design system](https://design-system.w3.org/) contains accessible HTML/CSS templates and components for use when creating W3C websites.
+
+* [Submit an issue about the Design System](https://github.com/w3c/w3c-website/issues/new?assignees=&labels=design+system&template=design_system.md&title=)
+
+GitHub repo: https://github.com/w3c/w3c-website-templates-bundle/
+
+## Craft CMS
+
+We use CraftCMS to manage content for the main W3C website.
```
Comment (vivienlacourba) [inline README.md:31]:
```suggestion
We use [Craft CMS](https://craftcms.com/) to manage content for the main W3C website.
```

**Review by vivienlacourba — CHANGES_REQUESTED** [review]
@simonrjones this looks great. Thank you. I only left one minor comment to link to the Craft website.

**Comment by simonrjones** [comment]
Good idea, added that link.

**Review by vivienlacourba — APPROVED** [review]

**Pull request merged**